DigitalLearningSolutions.Web/Controllers/SetPassword/ResetPasswordController.cs

```diff
@@ -1,6 +1,7 @@
 namespace DigitalLearningSolutions.Web.Controllers.SetPassword
 {
     using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;

```

Comment:
You can remove this, `UserService` is under `DigitalLearningSolutions.Data.Services`